libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxPrimaryManeuver.kt

```diff
@@ -43,31 +51,24 @@ class MapboxPrimaryManeuver : AppCompatTextView {
     constructor(
         context: Context,
         attrs: AttributeSet?,
-        defStyleAttr: Int
-    ) : super(context, attrs, defStyleAttr)
-
-    private var leftDrawable = ContextCompat.getDrawable(
-        context, R.drawable.mapbox_ic_exit_arrow_left
-    )
-    private var rightDrawable = ContextCompat.getDrawable(
-        context, R.drawable.mapbox_ic_exit_arrow_right
-    )
-    private var exitBackground = ContextCompat.getDrawable(
-        context, R.drawable.mapbox_exit_board_background
-    )
+        defStyleAttr: Int,
+        options: ManeuverPrimaryOptions = ManeuverPrimaryOptions.Builder().build()
+    ) : super(context, attrs, defStyleAttr) {
+        this.options = options
+    }
 
     /**
      * Invoke the method to render primary maneuver instructions
      * @param maneuver PrimaryManeuver
      */
     @Deprecated(
-        message = "The method can only render one shield if an instruction has multiple shields",
+        message = "The method may or may not render multiple shields for a given instruction",
```

Comment:
nit: kind of confused by the may or may not (seems like it cannot handle multiple)

```suggestion
        message = "The method does not handle multiple route shields provided by RouteShieldCallback",
```

Reply:
No, it's not true that this API cannot handle multiple. In some instances it can, whereas in some it might not.
@kmadsen